### PR TITLE
Update French translation

### PR DESCRIPTION
**Pull request by yannicka**
Some additional translations.

PS: thanks for Planner, I love it.

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -141,18 +137,21 @@ msgstr "Demain"
...
 "you need to do."
 msgstr ""
+"Ce projet vous aide à apprendre les bases de Planner et à démarrer avec un "
+"système de gestion de tâches simple pour rester organisé et au courant de "
+"tout ce que vous devez faire."
 
 #: src/Utils.vala:540
 msgid "Keeping track of your tasks"
-msgstr ""
+msgstr "Garder une trace de vos tâches"
```
Comment (NathanBnm) [inline po/fr.po]:
```suggestion
msgstr "Gardez un œil sur vos tâches"
```

File: po/fr.po [hunk 2]
```diff
@@ -238,14 +266,12 @@ msgid "Inbox"
 msgstr "Boîte de réception"
 
 #: src/Services/Todoist.vala:1197
-#, fuzzy
 msgid "Create todoist project error"
-msgstr "Créer un projet de tutoriel"
+msgstr "Erreur de création de projet Todoist"
```
Comment (NathanBnm) [inline po/fr.po]:
```suggestion
msgstr "Erreur de création du projet Todoist"
```

File: po/fr.po [hunk 3]
```diff
@@ -346,28 +378,30 @@ msgstr "Ajouter une liste"
...
+msgstr "Mode hors ligne activé"
 
 #: src/Widgets/Pane.vala:389
 msgid ""
 "Looks like you'are not connected to the\n"
 "internet. Changes you make in offline\n"
 "mode will be synced when you reconnect"
 msgstr ""
+"Il semblerait que vous ne soyez pas connecté\n"
+"à internet. Les changements que vous\n"
```
Comment (NathanBnm) [inline po/fr.po:390]:
```suggestion
"à Internet. Les changements que vous\n"
```
@alainm23 you should capitalize the word 'Internet'.

File: po/fr.po [hunk 4]
```diff
@@ -346,28 +378,30 @@ msgstr "Ajouter une liste"
...
 
 #: src/Widgets/New.vala:42
-#, fuzzy
 msgid "Name:"
-msgstr "Nom"
+msgstr "Nom :"
 
 #: src/Widgets/New.vala:46
 msgid "Source:"
 msgstr "Source:"
```
Comment (NathanBnm) [inline po/fr.po]:
```suggestion
msgstr "Source :"
```
Space is missing

File: po/fr.po [hunk 5]
```diff
@@ -382,62 +416,56 @@ msgid "Cancel"
...
 #: src/Widgets/New.vala:444
-#, fuzzy
 msgid "Project"
-msgstr "Nouveau projet"
+msgstr "Projet"
 
 #: src/Widgets/New.vala:450
 msgid "Start a new project, create a to do list, organize your notes."
 msgstr ""
+"Démarrer un nouveau projet, créer une liste de choses à faire, organisez vos "
```
Comment (NathanBnm) [inline po/fr.po]:
```suggestion
"Démarrez un nouveau projet, créez une liste de choses à faire, organisez vos "
```

File: po/fr.po [hunk 6]
```diff
@@ -382,62 +416,56 @@ msgid "Cancel"
...
+"notes."
 
 #: src/Widgets/New.vala:475
 msgid "Area"
 msgstr "Groupe"
 
 #: src/Widgets/New.vala:481
 msgid "Organize your projects in groups and keep your panel clean."
-msgstr "Organisez vos projets en groupes et gardez votre panneau propre."
+msgstr "Organiser vos projets en groupes et garder votre panneau propre."
```
Comment (NathanBnm) [inline po/fr.po]:
```suggestion
msgstr "Organisez vos projets en groupes et gardez votre panneau propre."
```

File: po/fr.po [hunk 7]
```diff
@@ -382,62 +416,56 @@ msgid "Cancel"
...
-#, fuzzy
 msgid "Move to area"
-msgstr "Convertir en projet"
+msgstr "Déplacer vers le groupe"
 
 #: src/Widgets/ProjectRow.vala:427
-#, fuzzy
 msgid "Project area"
-msgstr "Nouveau projet"
+msgstr "Projet de groupe"
```
Comment (NathanBnm) [inline po/fr.po]:
```suggestion
msgstr "Groupe de projet"
```

File: po/fr.po [hunk 8]
```diff
@@ -482,41 +508,36 @@ msgid "Delete task"
...
-#, fuzzy
 msgid "No Section"
-msgstr "Nouvelle section"
+msgstr "Pas de section"
 
 #: src/Widgets/ItemRow.vala:1126
-#, fuzzy
 msgid "Complete"
-msgstr "Tâches complétées"
+msgstr "Terminée"
```
Comment (NathanBnm) [inline po/fr.po]:
```suggestion
msgstr "Marquer comme terminé"
```
Depending of the context

File: po/fr.po [hunk 9]
```diff
@@ -532,7 +553,6 @@ msgid "Date completed"
 msgstr "Masquer les tâches complétées"
```
Comment (NathanBnm) [inline po/fr.po:553]:
```suggestion
msgstr "Date de fin"
```

Comment (NathanBnm) [reply po/fr.po:553]: I can't suggest it in this review but you have to remove the fuzzy attribute above.

File: po/fr.po [hunk 10]
```diff
@@ -591,33 +606,28 @@ msgid "Type a label"
 msgstr "Créer une étiquette"
```
Comment (NathanBnm) [inline po/fr.po]: Please remove the fuzzy attribute above.

File: po/fr.po [hunk 11]
```diff
@@ -591,33 +606,28 @@ msgid "Type a label"
...
-#, fuzzy
 msgid "Delete projects"
-msgstr "Supprimer le projet"
+msgstr "Supprimer les projets"
 
 #: src/Widgets/NewCheck.vala:53
-#, fuzzy
 msgid "Subtask name"
-msgstr "Nom de la tâche"
+msgstr "Nom de la sous-stâche"
```
Comment (NathanBnm) [inline po/fr.po]:
```suggestion
msgstr "Nom de la sous-tâche"
```

File: po/fr.po [hunk 12]
```diff
@@ -719,9 +728,8 @@ msgid "Inbox comments"
 msgstr "Commentaires de la boîte de réception"
 
 #: src/Views/Inbox.vala:80 src/Views/Project.vala:119
-#, fuzzy
 msgid "Search task"
-msgstr "Chercher la tâche"
+msgstr "Chercher une tâche"
```
Comment (NathanBnm) [inline po/fr.po]:
```suggestion
msgstr "Rechercher une tâche"
```

File: po/fr.po [hunk 13]
```diff
@@ -863,30 +861,28 @@ msgstr ""
...
 
 #: src/Dialogs/Preferences.vala:407
 msgid "AM/PM"
-msgstr ""
+msgstr "AM/PM"
 
 #: src/Dialogs/Preferences.vala:408
 msgid "24 h"
-msgstr ""
+msgstr "24 h"
```
Comment (NathanBnm) [inline po/fr.po:884]:
```suggestion
msgstr "24h"
```

File: po/fr.po [hunk 14]
```diff
@@ -927,18 +925,16 @@ msgstr ""
...
 #: src/Dialogs/Preferences.vala:663
-#, fuzzy
 msgid "Suggest Translations"
-msgstr "Traductions:"
+msgstr "Suggérer des traductions"
 
 #: src/Dialogs/Preferences.vala:664
 msgid "Fund"
-msgstr ""
+msgstr "Financement"
```
Comment (NathanBnm) [inline po/fr.po]:
```suggestion
msgstr "Soutenir"
```
Depending of the context

**Review by NathanBnm — CHANGES_REQUESTED** [review]
I left some comments, but here are some others comments I couldn't place in my review:
The following string has not been translated. I can't find context for it and literal translation won't fit well.
```
#: src/Widgets/Pane.vala:52
msgid "Back-Pocket"
msgstr "" 
```
Here a space is missing before the semicolon.
```diff
#: src/Widgets/ReminderButton.vala:172
msgid "Time:"
-msgstr "Heure:"
+msgstr "Heure :"
```
This string is marked as fuzzy. What format should it be?
```
#: src/Widgets/Calendar/CalendarHeader.vala:34
#, fuzzy
msgid "%OB, %Y"
msgstr "%e %B %Y"
```
The colon is missing to the following strings:
```diff
#: src/Views/Welcome.vala:29
msgid "Start working locally."
-msgstr "Commencez à travailler localement"
+msgstr "Commencez à travailler localement."
```
```diff
#: src/Views/Welcome.vala:30
msgid "Synchronize with your Todoist account and start working."
-msgstr "Synchronisez avec votre compte Todoist et commencez à travailler"
+msgstr "Synchronisez avec votre compte Todoist et commencez à travailler."
```
This translation sounds better in French (and the colon is missing):
```diff
#: src/Dialogs/TodoistOAuth.vala:83
msgid "Synchronizing… Wait a moment please."
-msgstr "Synchronisation… Attendez un instant s'il vous plaît"
+msgstr "Synchronisation… Veuillez patienter un instant."
```
Translation suggestion here too:
```diff
#: src/Dialogs/TodoistOAuth.vala:95
msgid "Please enter your credentials…"
-msgstr "Veuillez saisir vos informations d'identification…"
+msgstr "Veuillez saisir vos identifiants…"
```
The app name should not be translated:
```diff
#: src/Dialogs/Preferences.vala:348
msgid ""
"Personalize the look and feel of your Planner by choosing the theme that "
"best suits you."
msgstr ""
-"Personnalisez l'apparence de votre planificateur en choisissant le thème qui "
-"vous convient le mieux."
+"Personnalisez l'apparence de Planner en choisissant le thème qui "
+"vous convient le mieux."
```

Plus you forgot to translate the changelog.

File: po/fr.po (hunk 13 above)
Comment (yannicka) [reply po/fr.po:884]: The space is correct

File: po/fr.po (hunk 3 above)
Comment (yannicka) [reply po/fr.po:390]: "internet" may or may not be capitalized. There is no reason to capitalize it.

File: po/fr.po (hunk 9 above)
Comment (yannicka) [reply po/fr.po:553]: I didn't translate this string because I'm not sure about the translation.

**Comment by yannicka** [comment]
I took your proposals into account, thank you.

File: po/fr.po (hunk 3 above)
Comment (NathanBnm) [reply po/fr.po:390]: "Internet" should be capitalized if we talk about "the Internet" but not if used as an adjective like "site internet" in French.

File: po/fr.po (hunk 9 above)
Comment (NathanBnm) [reply po/fr.po:553]: I suggest "Date de fin", because "Date de complétion" sounds bad

File: po/fr.po (hunk 3 above)
Comment (yannicka) [reply po/fr.po:390]: "internet" *may* be capitalized, not *should*.

Comment (yannicka) [reply po/fr.po:390]: I think that discussing it here is useless and will not help. Nobody is wrong, nobody is right. Don't hesitate to do a PR if you think the use of "internet" without capitalization is incorrect.

**Comment by alainm23** [comment]
Thank you so much guys.

**Pull request merged**